### PR TITLE
Fix link in GFIs to point to no-assignee issues

### PR DESCRIPTION
**Pull request by seanlip**
This fixes the good-first-issues link in the starting instructions to point to issues that have no assignees.

### CODE REVIEW

**Comment by seanlip** [comment]
@lkbhitesh07 PTAL? Thanks!

File: Contributing-code-to-Oppia.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ We also **strongly recommend** looking through the resources under "Developing O
...
 
-After that, you can choose a good first issue from the [list of good first issues](https://github.com/oppia/oppia/labels/good%20first%20issue). These issues are hand-picked to ensure that you don't run into unexpected roadblocks while working on them, and each of them should have clear instructions for new contributors. If you see one that doesn't, please let us know via [GitHub Discussions](https://github.com/oppia/oppia/discussions) and we'll fix it. For other issues, you might need to be more independent because we might not know how to solve them either.
+After that, you can choose a good first issue from the [list of unassigned good first issues](https://github.com/oppia/oppia/issues?q=is%3Aopen+label%3A%22good+first+issue%22+no%3Aassignee). These issues are hand-picked to ensure that you don't run into unexpected roadblocks while working on them, and each of them should have clear instructions for new contributors. If you see one that doesn't, please let us know via [GitHub Discussions](https://github.com/oppia/oppia/discussions) and we'll fix it. For other issues, you might need to be more independent because we might not know how to solve them either.
```
Comment (Ash-2k3) [inline Contributing-code-to-Oppia.md:79]: Providing the link to unassigned issues is a good idea. However, I had a thought in mind: there are often cases where a new contributor gets assigned to an issue but then disappears. For e.gh: https://github.com/oppia/oppia/issues/20564.

So, I think it's fine to show such issues to them.

Just wanted to check this thought with you!

**Review by Ash-2k3 — COMMENTED** [review]
Hi @seanlip, I have left a comment, PTAL!

File: Contributing-code-to-Oppia.md (hunk 1 above)
Comment (seanlip) [reply Contributing-code-to-Oppia.md:79]: Hi @Ash-2k3, that's a great catch, thanks!

I think this is not the place to fix that, though. We already have multiple cases where new contributors try to claim issues that have already been assigned. Instead, we should have some way to check for stale assignments and drop those. I will file an issue to get oppiabot to do that.

Comment (seanlip) [reply Contributing-code-to-Oppia.md:79]: Filed https://github.com/oppia/oppia/issues/21294

**Review by Ash-2k3 — APPROVED** [review]
Thanks @seanlip, LGTM!

File: Contributing-code-to-Oppia.md (hunk 1 above)
Comment (Ash-2k3) [reply Contributing-code-to-Oppia.md:79]: Looks good, thanks!

**Pull request merged**